napari/_qt/_qapp_model/qactions/_help.py

```diff
@@ -0,0 +1,41 @@
+"""This module defines help menu actions.
+"""
+from typing import List
+
+from app_model.types import Action, KeyBindingRule, KeyCode, KeyMod
+
+from ...._app_model.constants import CommandId, MenuId
+from ....utils.translations import trans
+from ...dialogs.qt_about import QtAbout
+from ...qt_main_window import Window
+
+try:
+    from napari_error_reporter import ask_opt_in
+except ModuleNotFoundError:
+    ask_opt_in = None
+
+
+def _show_about(window: Window):
+    QtAbout.showAbout(window._qt_window)
+
+
+HELP_ACTIONS: List[Action] = [
```

Comment:
For consistency could this be `Q_HELP_ACTIONS`, see: https://github.com/napari/napari/pull/4977#discussion_r980615906